bin/i18n/resources/apps/text_to_speech/sync_out.rb

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+require_relative '../../../../../dashboard/config/environment'
+require_relative '../../../../../dashboard/app/models/concerns/text_to_speech'
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_out_base'
+require_relative '../text_to_speech'
+
+module I18n
+  module Resources
+    module Apps
+      module TextToSpeech
+        class SyncOut < I18n::Utils::SyncOutBase
+          METRIC_CONTEXT = 'update_i18n_static_messages'.freeze
+
+          LABS_FEEDBACK_MESSAGE_KEYS = {
```

Comment:
nit; I would like for us to carry over the TODO from the old script; in the long run, we really do want to be able to flag these values in apps code rather than manually maintaining this list here. No matter how much better this script is now than it was before!